src/libtrx/game/phase/executor.c

```diff
@@ -15,6 +15,7 @@
 #include "game/shell.h"
 #include "game/ui.h"
 #include "gfx/gl/track.h"
+#include "vector.h"
```

Comment:
Looks like this isn't needed.

Reply:
Thanks. Let's work on the audio drift in a separate ticket.